src/test/java/com/github/kokorin/jaffree/ffmpeg/FFmpegTest.java

```diff
@@ -240,6 +241,29 @@ public void onProgress(FFmpegProgress progress) {
                 .execute();
     }
 
+    @Test
+    public void testFrameCountingWithStreamCopyAndProgressListener() throws Exception {
+        final AtomicReference<Long> frameRef = new AtomicReference<>();
+
+        final ProgressListener progressListener = new ProgressListener() {
+            @Override
+            public void onProgress(FFmpegProgress progress) {
+                System.out.println(progress);
+                frameRef.set(progress.getFrame());
+            }
+        };
+
+        final FFmpegResult result = FFmpeg.atPath(Config.FFMPEG_BIN)
+                .addInput(UrlInput.fromPath(Artifacts.VIDEO_NUT))
+                .addOutput(new NullOutput())
+                .setProgressListener(progressListener)
+                .execute();
+
+        // Fails in FFmpeg 6.1, 6.1.1, 6.1.2, 7.0, 7.0.1, 7.0.2. To be fixed in the next release, see:
+        // https://github.com/FFmpeg/FFmpeg/commit/598f541ba49cb682dcd74e86858c9a4985149e1f
+        assertNotNull(frameRef.get());
```

Comment:
Is it possible to detect `ffmpeg` version in use and skip the test based on that?

Reply:
Yes, I've added a simple OutputListener which checks if the version starts with either 6.1 or 7.0 and skips the test based on that.